web_monitoring/diffing_server.py

```diff
@@ -3,14 +3,26 @@
 import hashlib
 from importlib import import_module
 import inspect
-import json
 import tornado.gen
 import tornado.httpclient
 import tornado.ioloop
 import tornado.web
 import web_monitoring
 
 
+# MAP tokens in the REST API to functions in modules.
+# The modules do not have to be part of the web_monitoring package.
+CONFIG = {
+    "length": ["web_monitoring.differs", "compare_length"],
+    "identical_bytes": ["web_monitoring.differs", "identical_bytes"],
+    "pagefreezer": ["web_monitoring.differs", "pagefreezer"],
+    "side_by_side_text": ["web_monitoring.differs", "side_by_side_text"],
+    "html_text_diff": ["web_monitoring.differs", "html_text_diff"],
+    "html_source_diff": ["web_monitoring.differs", "html_source_diff"],
+    "html_visual_diff": ["web_monitoring.differs", "html_diff_render"]
+}
```

Comment:
Any reason not to dump the strings and just make the values actual references here?

```py
{
  "length": web_monitoring.differs.compare_length,
  # ...etc...
}
```

If this is Python code, that seems both more convenient (can remove `load_config()`) and exposes our dependencies to anything that can analyze Python.

Reply:
Yeah, I guess the utility of this whole scheme is gone now that we're not using a config file.